Tests/FluentAssertions.Specs/Common/TypeExtensions.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace FluentAssertions.Specs.Common
+{
+    internal static class TypeExtensions
```

Comment:
Are these methods duplicated in core library or only present i test project?

Reply:
They are duplicated. IMO, there's no need for the specs project to reuse internal code of the core project.

Reply:
My potential concern is if the methods get out-of-sync and are no longer 1:1 duplicates.

Reply:
For the purpose of the tests, they don't have to be in sync. Reusing them is IMO an example of overusing DRY.